docs/source/contributing/redux.md

```diff
@@ -59,81 +64,111 @@ or [for Firefox](https://addons.mozilla.org/en-US/firefox/addon/reduxdevtools/).
-the backend and available as props.
+enable proper {term}`server-side rendering` of components.
+
+Using it makes sure that the component will be constructed with the proper data
+already fetched from the backend and available as props.
 
 ```{note}
-Beware! The `asyncConnect` is available only to components that are
-attached directly to the router or its children. There are some components
-that decide their "rendering path" at render time, so this prohibits the
-use of asyncConnect in that component tree. The biggest example of this is
-{file}`src/components/theme/View/View.jsx` which decides on the render component based
-on
-inspecting the content, so it is not possible to use asyncConnect in any
-view/layout component!
+Beware! The `asyncConnect` is available only to components that are attached
+directly to the router or its children.
+
+There are some components that decide their "rendering path" at render time, so
+this prohibits the use of asyncConnect in that component tree.
+
+The biggest example of this is {file}`src/components/theme/View/View.jsx` which
+decides on the render component based on inspecting the content, so it is not
+possible to use asyncConnect in any view/layout component!
 ```
 
 Notice the `emailNotification` action being passed to `connect` in the above
-example. All action (which trigger global state updates) need to be passed as
-props by `connect`. You can't properly trigger an action unless you access it
-as a prop, for example `this.props.emailNotification()`. For Function
-Components you can use the `useDispatch` hook.
+example.
+
+All actions (which trigger global state updates) need to be passed as props by
+`connect`.
+
+You can't properly trigger an action unless you access it as a prop, for example
+`this.props.emailNotification()`.
+
+For Function Components you can use the `useDispatch` hook.
 
 Global state update fetches are typically triggered by components in the mount
-lifecycle stage. See for example `src/components/theme/Search/Search.jsx` for
-a component that needs to interact with the backend to show its content. In the
-Redux flow of information, actions trigger the asynchronous processes and when
-that content arrives to the global app, it is pushed as props through the
-`connect` mechanism. So components only deal indirectly with async information:
-they trigger getting that information and it will arrive as a property once it
-is ready.
+lifecycle stage.
+
+See for example `src/components/theme/Search/Search.jsx` for a component that
+needs to interact with the backend to show its content.
+
+In the Redux flow of information, actions trigger the asynchronous processes
+and when that content arrives to the global app, it is pushed as props through
+the `connect` mechanism.
+
+So components only deal indirectly with async information: they trigger getting
+that information and it will arrive as a property once it is ready.
```

Comment:
```suggestion
See for example {file}`src/components/theme/Search/Search.jsx` for a component that needs to interact with the backend to show its content.
In the Redux flow of information, actions trigger the asynchronous processes, and when that content arrives to the global app, it is pushed as props through the `connect` mechanism.
So components only deal indirectly with async information: they trigger getting that information and it will arrive as a property once it is ready.
```